examples/nats/go.mod

```
@@ -0,0 +1,54 @@
+module github.com/testcontainers/testcontainers-go/examples/nats
+
+go 1.20
+
+require (
+	github.com/nats-io/nats.go v1.25.0
+	github.com/testcontainers/testcontainers-go v0.19.0
+	gotest.tools/gotestsum v1.10.0
+)
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
+	github.com/Microsoft/go-winio v0.5.2 // indirect
+	github.com/cenkalti/backoff/v4 v4.2.0 // indirect
+	github.com/containerd/containerd v1.6.19 // indirect
+	github.com/cpuguy83/dockercfg v0.3.1 // indirect
+	github.com/dnephin/pflag v1.0.7 // indirect
+	github.com/docker/distribution v2.8.1+incompatible // indirect
```

Could you include this change, including running `go mod tidy` from the dir of the module? There was a dependabot update this morning including this security fix:
```suggestion
	github.com/docker/distribution v2.8.2+incompatible // indirect
```

---

`go mod tidy` seemed to not change anything so opted to use `go get -d -u -t ./...` to upgrade all deps.